src/plugins/controls/public/react_controls/control_group/init_controls_manager.test.ts

```diff
@@ -187,3 +189,51 @@ describe('getLastUsedDataViewId', () => {
     expect(dataViewId).toBeUndefined();
   });
 });
+
+describe('getNewControlState', () => {
+  test('should contain defaults when there are no existing controls', () => {
+    const controlsManager = initControlsManager({}, DEFAULT_DATA_VIEW_ID);
+    expect(controlsManager.getNewControlState()).toEqual({
+      grow: true,
+      width: 'medium',
+      dataViewId: DEFAULT_DATA_VIEW_ID,
+    });
+  });
+
+  test('should start with defaults if there are existing controls', () => {
+    const controlsManager = initControlsManager(
+      {
+        alpha: {
+          type: 'testControl',
+          order: 1,
+          dataViewId: 'myOtherDataViewId',
+          width: 'medium',
+          grow: true,
```

Comment:
Sorry woops - your fix made me realize I put the changes in the wrong location in my suggestion LOL - accidentally changed the initial control and not the `expect` state 🙈 We should have kept these to be something different than the default so that the test is actually... testing something hehe

```suggestion
          width: 'small',
          grow: false,
```